proxy/src/http/org.rs

```diff
@@ -0,0 +1,256 @@
+//! Endpoints for Org.
+
+use serde::ser::SerializeStruct as _;
+use serde::{Deserialize, Serialize, Serializer};
+use std::sync::Arc;
+use tokio::sync::RwLock;
+use warp::document::{self, ToDocumentedType};
+use warp::{path, Filter, Rejection, Reply};
+
+use crate::avatar;
+use crate::notification;
+use crate::registry;
+
+/// The org with identifier and avatar
+pub struct Org {
+    /// The unique identifier of the org
+    id: String,
+    /// Generated fallback avatar 
+    avatar_fallback: avatar::Avatar
+}
+
+/// Combination of all org routes.
+pub fn filters(
+    registry: Arc<RwLock<registry::Registry>>,
+    subscriptions: notification::Subscriptions,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    get_filter().or(register_filter(registry, subscriptions))
+}
+
+/// POST /orgs/register
+fn register_filter(
+    registry: Arc<RwLock<registry::Registry>>,
+    subscriptions: notification::Subscriptions,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    path!("orgs" / "register")
+        .and(warp::post())
+        .and(super::with_registry(registry))
+        .and(warp::body::json())
+        .and(super::with_subscriptions(subscriptions))
```

Comment:
I tend to order the params for the handler with datastores first and inputs, params last. So would swap line 38 and 39.